src/Components/display.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = props => {
+  const { result } = props;
+  return (
+    <div className="display">
+      <span>{result}</span>
+    </div>
+  );
+};
+Display.propTypes = {
+  result: PropTypes.number.isRequired,
```

Comment:
this should be a string ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)

Reply:
ahh thanks :)